src/main/java/com/minecolonies/api/colony/requestsystem/requestable/Tool.java

```diff
@@ -184,24 +186,16 @@ public ItemStack getResult()
     @Override
     public boolean matches(@NotNull final ItemStack stack)
     {
-        //API:Map the given strings a proper way.
-        final boolean toolTypeResult = !ItemStackUtils.isEmpty(stack)
+        return !ItemStackUtils.isEmpty(stack)
```

Comment:
can we untangle this stream to a loop? This is called fairly often and the list is not long, so the performance impact of this is for sure real